recipe/deploy/vendors.php

```diff
@@ -9,5 +9,11 @@
 
 desc('Installing vendors');
 task('deploy:vendors', function () {
+    $releases = get('releases_list');
+    if (isset($releases[1])) {
+        if (run("if [ -d {{deploy_path}}/releases/{$releases[1]}/vendor ]; then echo 'true'; fi")->toBool()) {
```

Comment:
Aside from what was said: `vendor-dir` is configurable and thus needs to be read from Composer config.